packages/combo-box/test/lazy-loading.test.js

```diff
@@ -813,7 +813,7 @@ describe('lazy loading', () => {
         it('should render selectedItem as selected when setting it while loading more items', async () => {
           const alreadyLoadedItem = `item ${comboBox.pageSize - 1}`;
 
-          scrollToIndex(comboBox, comboBox.pageSize + 1);
+          scrollToIndex(comboBox, comboBox.pageSize - 1);
```

Comment:
Scrolling to `index + 1` guarantees the item `index + 1` ends up inside the visible viewport, but it this may result in the item `index - 1` ending outside of it. Items outside the viewport may or may not get rendered to the DOM. Changed the test to scroll to `index - 1` to make sure it's actually available in the DOM.